policyengine_us/variables/gov/states/la/tax/income/exempt_income/la_disability_income_exemption_person.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class la_disability_income_exemption_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Louisiana disability income exemption for each person"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.legis.la.gov/legis/Law.aspx?d=102133"  # (B)
+    defined_for = StateCode.LA
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.la.tax.income.exempt_income.disability
+        # All disability benefits are exempt from income tax
```

Comment:
You got it backwards (but again could you please confirm?)
```suggestion
        # The disability_benefits variable includes only taxable disability benefits.
```

Reply:
The law points to the following Louisiana [section](https://law.justia.com/codes/louisiana/2022/revised-statutes/title-23/rs-23-1221/): this states permanent total disability which restricts the person from employment (2)
However, it does specify all disability benefits:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/a5568a54-7c85-4ed0-93ca-16c4fc4f67b4)